content/rs/developing/crdbs/_index.md

```diff
@@ -191,6 +191,13 @@ the syncer initiates full reconciliation with other peers in the CRDB.
 
 ## INFO
 
+The INFO command shows these statistics differently for CRDBs:
+
+- DBSIZE (in `shard-cli info`) reports key header instances, not unique keys.
```

Comment:
DBSIZE is a command, it's not in the info string
maybe it's worth explaining what are key headers:
if replica1 did 'SET  x 1` and replica2 did `SADD x 1' simultaneously (i.e. connection between them was dropped) and then connection is restored, DBSIZE will show 2 but x is visibly a string on all replicas (in case of type conflicts such as that, string wins set)

Reply:
How about: DBSIZE (in `shard-cli dbsize`) reports key header instances that represent multiple potential values of a key before a replication conflict is resolved.